backend/authentication/adminPasswordConfirm.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+const AWS = require('aws-sdk');
+const db = new AWS.DynamoDB.DocumentClient();
+const uuid = require('uuid');
+require('dotenv').config();
+
+async function adminPasswordConfirm(event, context, callback) {
+    try {
+        // Get the recovery code from db for this user
+        const recoveryCodes = await db.scan({
+            TableName: process.env.RECOVERY_CODES_TABLE
+        }).promise();
+
+        let recoveryEntry;
+        for (let i = 0; i < recoveryCodes.Items.length; i++) {
+            if (recoveryCodes.Items[i].code == event.arguments.confirmationCode) {
```

Comment:
Extract the variables from `event.arguments` into constants and use them instead. It's quite nasty to write every time event.arguments.
A good approach is `const { propertyName } = event.arguments`